tests/console/yast2_xinetd.pm

```diff
@@ -27,7 +27,11 @@ sub run {
     script_run("yast2 xinetd; echo yast2-xinetd-status-\$? > /dev/$serialdev", 0);
 
     # check xinetd network configuration got started
-    assert_screen 'yast2_xinetd_startup', 90;
+    assert_screen([qw(yast2_xinetd_startup yast2_xinetd_core-dumped)]), 90;
+    if (match_has_tag('yast2_xinetd_core-dumped')) {
+        # softfail when yast2 crashed and throws core dumped message
+        record_soft_failure "bsc#1049433";
```

Comment:
just add `return 1;` to make the whole test step soft fail and *not* fail. But if you do this then no logs are collected or uploaded. Better to call the post_fail_hook explicitly and then return, e.g. (untested):

Add in line 20:

```perl
my $self = @_;
```

```perl
record_soft_failure "bsc#1049433";
$self->SUPER::post_fail_hook();
return 1;
```

you can test that with a test needle that matches even normal passing windows but the tag "yast2_xinetd_core-dumped".

Reply:
@Zaoliang could you please update? ;)